README.rst

```diff
@@ -27,7 +27,7 @@ Compatibility Notes
 +==================+==============+
 | Palm             | >= 0.5.0     |
```

Comment:
We need to add the limit here as well. This version also looses compatibility with Palm